restangular/restangular.d.ts

```diff
@@ -116,6 +116,9 @@ declare namespace restangular {
     withHttpConfig(httpConfig: angular.IRequestShortcutConfig): IElement;
     save(queryParams?: any, headers?: any): IPromise<any>;
     getRestangularUrl(): string;
+    route?: string;
+    id?: string;
```

Comment:
@vvakame is `string` correct type for `id`, shouldn't this be `number` ?

Reply:
It was not changed to support `number`, while you seems to agree it should. I am making a PR